cartographer/mapping/proto/internal/legacy_probability_grid.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2016 The Cartographer Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package cartographer.mapping.proto;
+
+import "cartographer/mapping/proto/2d/map_limits.proto";
+
+// The legacy probability grid that was used before the generalized
+// grid structure was introduced.
+
+message LegacyProbabilityGrid {
```

Comment:
how about adding this to the `LegacySerializedData.proto`?

Reply:
I can also put both `LegacySubmap` and `LegacyProbabilityGrid` in there. But it doesn't really bother if we have 1, 2 or 3 files there, so how you like.

Reply:
Fine with me.